src/React.re

```diff
@@ -25,6 +25,8 @@ module Ref = {
   [@bs.set] external setCurrent: (t('value), 'value) => unit = "current";
 };
 
+[@bs.module "react"] external createRef: unit => Ref.t('a) = "";
```

Comment:
This needs to be `unit => Ref.t(Js.nullable('a))` or else I don't think it's safe.